src/components/EditCollapse.js

```diff
@@ -11,6 +11,24 @@ import ChangelistContext from './ChangelistContext';
 import CubeContext from './CubeContext';
 import CSRFForm from './CSRFForm';
 
+export const getCard = async (name) => {
+  if (name && name.length > 0) {
+    const normalized = encodeName(name);
+    const response = await fetch(`/cube/api/getcard/${normalized}`);
+    if (!response.ok) {
+      setAlerts((alerts) => [...alerts, { color: 'danger', message: `Couldn't get card: ${response.status}.` }]);
+      return null;
+    }
+
+    const json = await response.json();
+    if (json.success !== 'true' || !json.card) {
```

Comment:
In what situation will only one of these conditionals be true? Would it make sense to just use one if statement here?

Reply:
I don't think there is one, just trying to be robust to weird failures + bugs